code/__HELPERS/text.dm

```diff
@@ -438,3 +438,85 @@ var/list/binary = list("0","1")
 			ca -= 13
 		result += ascii2text(ca)
 	return jointext(result, "")
+
+//Takes a list of values, sanitizes it down for readability and character count,
+//then exports it as a plaintext file at data/npc_saves/[filename].txt.
+//As far as SS13 is concerned this is write only data. You can't change something
+//in the txt file and have it be reflected in the in game item/mob it came from.
+//(That's what things like savefiles are for) Note that this list is not shuffled.
+/proc/twitterize(list/proposed, filename, cullshort = 0, storemax = 1000)
+	if(!islist(proposed) || !filename || storemax > proposed.len)
+		return
+
+	var/list/nope = list("www.", "http", "@") //Note the ommission of #, hashtags are fine

```

Comment:
we should probably have a file with blacklisted words too for server owners if we want to blacklist slurs and shit easily